Lib/asyncio/taskgroups.py

```diff
@@ -29,8 +26,8 @@ def __repr__(self):
         info = ['']
         if self._tasks:
             info.append(f'tasks={len(self._tasks)}')
-        if self._unfinished_tasks:
-            info.append(f'unfinished={self._unfinished_tasks}')
+        if self._tasks:
+            info.append(f'unfinished={len(self._tasks)}')
```

Comment:
(trivial) this can be combined into the previous if block.

We're not dropping this `unfinished=` repr for compatibility concerns? If yes, do we need a `_all_tasks` counter and set `tasks={self._all_tasks}'` in previous line?

Reply:
Oh my, I didn't even notice that it's two duplicate blocks. Yeah, we don't need `unfinished=` anymore.

Reply:
Pushed an update

Reply:
Regarding counting the tasks -- I can see it being useful for maybe some long-running groups... I think if we want to add that it should be a separate PR, ideally requested by a user :)